DESCRIPTION

```diff
@@ -28,6 +28,7 @@ Imports:
     foghorn,
     gert (>= 1.0.0),
     gh,
+    git2r,
```

Comment:
We won't need this after #85.

Reply:
Yes, but the idea of this PR was to be merged before #85 to have a valid CI check for future work on `f-27-automate`